gp2gp-translator/src/main/java/uk/nhs/adaptors/pss/translator/task/scheduled/EHRTimeoutHandler.java

```diff
@@ -171,7 +171,7 @@ private void handleMigrationTimeout(PatientMigrationRequest migrationRequest, NA
             LOGGER.error("Error retrieving persist duration: [{}]", e.getMessage());
         } catch (JsonProcessingException | SAXException | DateTimeParseException | JAXBException e) {
             LOGGER.error("Error parsing inbound message from database");
-            migrationStatusLogService.addMigrationStatusLog(EHR_GENERAL_PROCESSING_ERROR, conversationId, null);
+            migrationStatusLogService.addMigrationStatusLog(EHR_GENERAL_PROCESSING_ERROR, conversationId, null, "99");
```

Comment:
What was the thinking behind "99" in this case. I'm not sure I have an alternative suggestion as to what we could do here, just curious as to the thinking.